webextensions/api/dom.json

```
@@ -11,7 +11,8 @@
               },
               "edge": "mirror",
               "firefox": {
-                "version_added": "63"
+                "version_added": "false",
+                "notes": "Use <code>element.openOrClosedShadowRoot</code> on Firefox."
```

```suggestion
                "notes": "Use <code>element.openOrClosedShadowRoot</code>, available since Firefox 63."
```

The loss of the version information is somewhat unfortunate. Here is a suggested edit to cover the addition (confirmed 63 - https://bugzilla.mozilla.org/show_bug.cgi?id=1439153 )